react-app/assets/exercises/breath.svg

```diff
@@ -0,0 +1,31 @@
+<svg width="100" height="193" viewBox="0 0 100 193" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are we using this svg in this PR? if not, let's remove it for now

Reply:
We can remove it for now, this one is not good enough.